src/sdkV2.ts

```diff
@@ -104,22 +112,150 @@ export default class CegaEvmSDKV2 {
     });
   }
 
-  async bulkSettleVaultsDcs(
+  async bulkProcessDepositQueuesDcs(
     vaultAddresses: EvmAddress[],
+    maxProcessCount: ethers.BigNumber,
+    overrides: TxOverrides = {},
   ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = this.loadCegaEntry();
-    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+    return cegaEntry.bulkProcessDCSDepositQueues(vaultAddresses, maxProcessCount, {
+      ...(await this._gasStation.getGasOraclePrices()),
+      ...overrides,
+    });
   }
 
+  async endAuctionDcs(
+    vaultAddress: EvmAddress,
+    auctionWinner: EvmAddress,
+    tradeStartDate: ethers.BigNumber,
+    aprBps: ethers.BigNumber,
+    oracleDataSource: OracleDataSourceDcs,
+    overrides: TxOverrides = {},
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.endDCSAuction(
+      vaultAddress,
+      auctionWinner,
+      tradeStartDate,
+      aprBps,
+      oracleDataSource,
+      {
+        ...overrides,
+      },
+    );
+  }
+
+  /**
+   * MARKET MAKER METHODS
+   */
+
   async bulkStartTradesDcs(
     vaultAddresses: EvmAddress[],
   ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = this.loadCegaEntry();
     return cegaEntry.bulkStartDCSTrades(vaultAddresses);
   }
 
+  async bulkSettleVaultsDcs(
+    vaultAddresses: EvmAddress[],
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+  }
+
+  /**
+   * SETTLEMENT METHODS
```

Comment:
CEGA SETTLEMENT METHODS

(because there's a "settle" method for the MMs to call also